Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -5185,6 +5184,27 @@ on revIDEPropertySet pObjectList, pProperty, pValue, pLockUpdates
       put true into sLockUpdates
    end if
    
+   # Set stack edited before property change
+   local tStacksListA, tStackShortNamesListA, tStack, tStackName
+   repeat for each line tObject in pObjectList
+      put revIDEStackOfObject(tObject) into tStack
+      if not tStacksListA[tStack] then
+         put true into tStacksListA[tStack]
+         
+         # If stack name is changed, use new name when setting edited
+         if tStack is tObject and pProperty is "name" then
+            put pValue into tStackName
+         else
+            put the short name of tStack into tStackName
+         end if
+         put true into tStackShortNamesListA[tStackName]
```

Comment:
@livecodeali When the name of the stack changes, is it possible to delete the old entry in `tStackShortNamesListA`?